pkg/bpm/bpm.go

```diff
@@ -63,11 +64,27 @@ const (
 )
 
 // ProcessPair is an identifier for process
+// Keep compatible with v2.x
+// TODO: remove in v3.x
 type ProcessPair struct {
 	Pid        int
 	CreateTime int64
 }
 
+type Process struct {
+	Uid string
+
+	// TODO: remove in v3.x
+	// store create time, to keep compatible with v2.x
+	Pair ProcessPair
+
+	cmd   *ManagedCommand
+	stdio *Stdio
+
+	ctx     context.Context
+	stopped func()
+}
+
 // Stdio contains stdin, stdout and stderr
 type Stdio struct {
```

Comment:
I think we can remove the Locker here now, chaos developers should add locker by themselves when they need it now.

Reply:
And then it is possible that we can just use the raw stdin out err.

Reply:
The lock should be stored in the bpm processes, otherwise, we cannot share the lock between parallel requests.